packages/react-components/src/components/chart/legend/table/columnDefinitions/factory.tsx

```diff
@@ -17,6 +19,15 @@ const createDataStreamColumnDefinition = (
   isRowHeader: true,
 });
 
+const createAssetNameColumnDefinition =
+  (): LegendTableColumnDefinitions[1] => ({
+    id: 'AssetName',
+    sortingField: 'assetName',
+    header: <AssetNameColumnHeader />,
+    cell: (item) => <AssetNameCell {...item} />,
+    isRowHeader: true,
+  });
+
```

Comment:
How does this work with unmodeled data streams?

Reply:
showing empty in unmodeled data streams
![image](https://github.com/awslabs/iot-app-kit/assets/142866907/da47d6ad-85f0-41eb-a91c-62257969bf18)